project.clj

```diff
@@ -12,15 +14,17 @@
   :pedantic? :abort
 
   :dependencies [[org.clojure/clojure "1.6.0"]
+                 [puppetlabs/trapperkeeper ~tk-version]
+                 [puppetlabs/trapperkeeper-webserver-jetty9 ~tk-jetty-version]
                  ;; Logging
                  [org.clojure/tools.logging "0.2.6"]
                  ;; Filesystem utilities
                  [me.raynes/fs "1.4.5"]
                  [org.clojure/tools.cli "0.3.0"]
-                 [prismatic/schema "0.2.2"]
+                 [prismatic/schema "0.4.0"]
                  [inet.data "0.5.5"]
                  [clj-time "0.5.1"]
-                 [puppetlabs/typesafe-config "0.1.1"]
+                 [puppetlabs/typesafe-config "0.1.4"]
                  [puppetlabs/ssl-utils "0.8.0"]]
```

Comment:
FYI we have `0.8.1` out, though it bumps the `clj-time` dependency to `0.7.0` and none of the other TK libs are on that version so we end up having to do some exclusions.